server/src/weaverbird/backends/pypika_translator/translators/base.py

```diff
@@ -128,6 +128,8 @@ class SQLTranslator(ABC):
     FROM_DATE_OP: FromDateOp
     REGEXP_OP: RegexOp
     TO_DATE_OP: ToDateOp
+    # depending on the translator, this may change to ` or '
+    QUOTE_CHAR = '"'
```

Comment:
```suggestion
    QUOTE_CHAR: str
```

Let's be explicit :smile: Having the `QUOTE_CHAR` defined in all connectors, like the rest of the attributes, will allow us to see errors more easily

Reply:
https://github.com/ToucanToco/weaverbird/pull/1425/commits/ab26fe1251ba3344c4df7244c6a6655c064d7065